docker-files-for-Gitlab-CI-rust/substrate/Dockerfile

```diff
@@ -1,31 +1,36 @@
-FROM parity/rust:gitlab-ci as builder 
-MAINTAINER Parity Technologies <devops@parity.io>
+FROM parity/rust-parity-ethereum-build:xenial as builder
```

Comment:
this can be changed to the new parity-eth builder: https://github.com/paritytech/scripts/blob/master/docker-files-for-Gitlab-CI-rust/parity-eth-linux-ci/Dockerfile